internals/detect_intent.py

```diff
@@ -90,6 +96,39 @@ def detect_thread_url(body):
   return None
 
 
+def detect_lgtm(body):
+  """Return true if LGTM is on first non-blank line."""
+  lines = body.split('\n')
+
+  first_nonblank_line = ''
+  for line in lines:
+    if line.strip():
+      first_nonblank_line = line.strip()
+      break
+
+  if (first_nonblank_line.startswith('>') or
+      NOT_LGTM_RE.search(first_nonblank_line)):
+    return False
+
+  match = LGTM_RE.search(first_nonblank_line)
+  return match
```

Comment:
I think the google style guide favors using truthy interpretations of values rather than explicitly converting to boolean. 
https://g3doc.corp.google.com/eng/doc/devguide/py/style/index.md?cl=head#truefalse-evaluations

The python docs for regexes also show using the match object as a truthy value, although that is in an if-statement rather than being returned.
https://docs.python.org/3/library/re.html#match-objects

Let me know if you still prefer making it explicit and I will.

Reply:
Yea that makes sense. Thanks for pointing it out